funding/manager.go

```diff
@@ -3248,7 +3249,7 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 	// the remote peer are signaling the proper feature bit.
 	commitType, err := negotiateCommitmentType(
 		msg.ChannelType, msg.Peer.LocalFeatures(),
-		msg.Peer.RemoteFeatures(),
+		msg.Peer.RemoteFeatures(), false,
```

Comment:
false -> true

Reply:
Flipped, I also flipped the bool when we process accept but they set a type (case #1 above), as we're ok w/ implicit fallback there. Thinking I might change the variable name there as well.